tau_embedding_settings.py

```diff
@@ -181,4 +194,823 @@ def setup_embedding(configuration: Configuration, scopes: List[str]):
+    )
+
+    # use other trigger flags for embedding samples
+    configuration.add_modification_rule(
+        "mt",
+        ReplaceProducer(
+            producers=[
+                triggers.MTGenerateCrossTriggerFlags,
+                triggers.MTGenerateCrossTriggerFlagsEmbedding,
+            ],
+            samples="embedding",
+        ),
+    )
+    configuration.add_outputs(
+        "mt", triggers.MTGenerateCrossTriggerFlagsEmbedding.output_group
+    )
+
+    ######################
+    ## Tau ID SFs
+    ######################
+
+    configuration.add_modification_rule(
+        ["et", "mt"],
+        ReplaceProducer(
+            producers=[
+                scalefactors.Tau_2_VsJetTauID_lt_SF,
+                embedding.Tau_2_VsJetTauID_lt_SF,
+            ],
+            samples="embedding",
+        ),
+    )
+    configuration.add_modification_rule(
+        "tt",
+        ReplaceProducer(
+            producers=[
+                scalefactors.Tau_1_VsJetTauID_SF,
+                embedding.Tau_1_VsJetTauID_tt_SF,
+            ],
+            samples="embedding",
+        ),
+    )
+    configuration.add_modification_rule(
+        "tt",
+        ReplaceProducer(
+            producers=[
+                scalefactors.Tau_2_VsJetTauID_tt_SF,
+                embedding.Tau_2_VsJetTauID_tt_SF,
+            ],
+            samples="embedding",
+        ),
+    )
+    configuration.add_outputs(
+        ["et", "mt"],
+        embedding.Tau_2_VsJetTauID_lt_SF.output_group,
+    )
+    configuration.add_outputs(
+        "tt",
+        [
+            embedding.Tau_1_VsJetTauID_tt_SF.output_group,
+            embedding.Tau_2_VsJetTauID_tt_SF.output_group,
+        ],
+    )
+
+    # replace TauID producers for embedding samples
+    configuration.add_config_parameters(
+        ["mt", "et"],
+        {
+            "tau_emb_sf_file": "data/embedding/tau_2018UL.json.gz",
+            "tau_emb_sf_vsjet_tau20to25": "nom",
+            "tau_emb_sf_vsjet_tau25to30": "nom",
+            "tau_emb_sf_vsjet_tau30to35": "nom",
+            "tau_emb_sf_vsjet_tau35to40": "nom",
+            "tau_emb_sf_vsjet_tau40toInf": "nom",
+            "tau_emb_id_sf_correctionset": "TauID_sf_embedding_ptbinned",
+            "vsjet_tau_id_sf_embedding": [
+                {
+                    "tau_1_vsjet_sf_outputname": "id_wgt_tau_vsJet_{wp}_1".format(
+                        wp=wp
+                    ),
+                    "tau_2_vsjet_sf_outputname": "id_wgt_tau_vsJet_{wp}_2".format(
+                        wp=wp
+                    ),
+                    "vsjet_tau_id_WP": "{wp}".format(wp=wp),
+                }
+                for wp in [
+                    # "VVVLoose",
+                    # "VVLoose",
+                    # "VLoose",
+                    # "Loose",
+                    # "Medium",
+                    "Tight",
+                    # "VTight",
+                    # "VVTight",
```

Comment:
Does this mean that there is only the tight WP or that the others are not needed?